.circleci/config.yml

```diff
@@ -135,114 +141,117 @@ references:
           }
         }'
 
-jobs:
-  build:
-    macos:
-      xcode: "9.0"
-    shell: /bin/bash --login
-    working_directory: /Users/distiller/wp-desktop
-    steps:
-      - checkout
-      - run:
-          name: Setup calypso
-          command: |
-                  git submodule init
-                  git submodule update
-
-                  if [ -n "${CALYPSO_HASH}" ]; then
-                    cd calypso;
-                    git fetch;
-                    git checkout ${CALYPSO_HASH};
-                  fi
-      - *calypso_prepare_cache
-      - *calypso_restore_cache
-      - *restore_nvm
-      - *setup_nvm
-      - *save_nvm
-      - run:
-          name: Decrypt assets
-          command: |
-                  openssl aes-256-cbc -d -in resource/calypso/secrets.json.enc -out calypso/config/secrets.json -k "${CALYPSO_SECRETS_ENCRYPTION_KEY}"
-                  openssl aes-256-cbc -d -in resource/certificates/mac.p12.enc -out resource/certificates/mac.p12 -k "${CALYPSO_SECRETS_ENCRYPTION_KEY}"
-                  openssl aes-256-cbc -d -in resource/certificates/win.p12.enc -out resource/certificates/win.p12 -k "${CALYPSO_SECRETS_ENCRYPTION_KEY}"
-      - *npm_restore_cache
-      - run:
-          name: Npm install
-          command: |
-                  source $HOME/.nvm/nvm.sh
-                  nvm use
-                  npm ci
-                  cd calypso
-                  npm ci
-      - *npm_save_cache
-      - run:
-          name: Build sources
-          no_output_timeout: 20m
-          command: |
-                  # only use the updater when we are building from master or a release branch
-                  # if [[ $CIRCLE_BRANCH =~ ^release(.*)|(^master$) ]]
-                  # then
-                  #   CONFIG_ENV=release
-                  # fi
 
-                  # TODO: update accordingly when code from above changes
-                  CONFIG_ENV=release
+orbs:
```

Comment:
> Orbs are packages of config that you can use to quickly get started with the CircleCI platform. Orbs enable you to share, standardize, and simplify config across your projects.

[Using Orbs](https://circleci.com/docs/2.0/using-orbs/)

Are we intending to share this at all, or is to reuse a set of steps?

For a locally reusable set of steps, we're using commands in wp-calypso:

https://github.com/Automattic/wp-calypso/blob/d28aab8c217669f1d3fbf7a9286df7506e3fec03/.circleci/config.yml#L144-L151

```yaml
commands:
  prepare:
    description: "Prepare Job"
    steps:
      # folders to collect results
      - run: *setup-results-and-artifacts
      - attach_workspace:
          at: '~'

# …

jobs:
  some-job:
    steps:
      - prepare
```

Would that be a better fit, or is there other motivation for using orbs?

Reply:
Oh, I didn't realize you could declare commands directly without an inline Orb. That seems a better fit.

Of course this still requires using CircleCI 2.1.

Reply:
Done. As a bonus, this makes the diff easier to read since there is now no indentation change.